staging/src/k8s.io/kubectl/pkg/cmd/config/create_cluster.go

```diff
@@ -165,12 +173,11 @@ func (o *createClusterOptions) complete(cmd *cobra.Command) error {
 	if len(args) != 1 {
 		return helpErrorf(cmd, "Unexpected args: %v", args)
 	}
-
 	o.name = args[0]
 	return nil
 }
 
-func (o createClusterOptions) validate() error {
+func (o *createClusterOptions) validate() error {
```

Comment:
Wdyt about setting `o.caData` earlier, perhaps in `func (o *createClusterOptions) complete`? I'm wary of a `validate` function actually mutating, and of L154 depending on a side-effect of validation being run to be effective.